src/js/login/containers/Main.jsx

```diff
@@ -141,7 +145,7 @@ class Main extends React.Component {
 const mapStateToProps = (state) => {
   const userState = state.user;
   return {
-    login: userState.login,
+    loginUrl: userState.loginUrl,
```

Comment:
The second option might be slightly clearer, but for the sake of getting this PR in, I think the first option still works well and we can always refactor that later :).